src/interfaces/node.cpp

```diff
@@ -335,6 +335,7 @@ class NodeImpl : public Node
         return GuessVerificationProgress(Params().TxData(), tip);
     }
     bool isInitialBlockDownload() override { return ::ChainstateActive().IsInitialBlockDownload(); }
+    bool isAddressTypeSet() override { return !::gArgs.GetArg("-addresstype", "").empty(); }
```

Comment:
16497: looks useless for us (we have no `-addresstype` and no bech32 addresses). maybe DNM?

Reply:
if that is the case should i also DNM #15711 on our sheet as it is the first set of changes? looks nothing more than a syntax change to call bech32 addresses `legacy addresses`

Reply:
yes, I guess so